ompi/mpi/man/man3/Makefile.am

```diff
@@ -77,7 +133,7 @@ MD_FILES = \
         MPI_Graph_get.md \
         MPI_Graph_map.md \
         MPI_Graph_neighbors_count.md \
-        MPI_Graphdims_get.md \
+        MPI_Graphdims_get.md \a
```

Comment:
```suggestion
        MPI_Graphdims_get.md \
```

Reply:
I thought I removed that, something funky must have been going on with my vim or adding the edited Makefile to the commit